metadata-ingestion/src/datahub/ingestion/transformer/base_transformer.py

```diff
@@ -28,7 +28,9 @@ def _update_work_unit_id(
 
 
 class HandleEndOfStreamTransformer:
-    def handle_end_of_stream(self) -> List[MetadataChangeProposalWrapper]:
+    def handle_end_of_stream(
+        self,
+    ) -> List[Union[MetadataChangeProposalWrapper, MetadataChangeProposalClass]]:
```

Comment:
this should let us simplify types in methods that override handle_end_of_stream

```suggestion
    ) -> Sequence[Union[MetadataChangeProposalWrapper, MetadataChangeProposalClass]]:
```